### PR TITLE
223 support request mileage

### PR DESCRIPTION
**Pull request by misternu**
## Changelog
- Enable gas mileage input and cost calculation on support request create/edit forms
- Unify support request form partial and update createsupportrequest to match

## Link to issue:  
#223 

## Relevant Screenshots: 
Screen Shot 2019-11-24 at 4 50 04 PM

## Are you ready for review?:

- [x] Added relevant tests
- [x] Linked PR to the issue
- [ ] Added notes for QA/special notes
- [x] Added revelant screenshots

### CODE REVIEW

**Comment by misternu** [comment]
Screen Shot 2019-12-01 at 5 13 37 PM

File: app/javascript/src/transactions.js [hunk 1]
```diff
@@ -11,6 +12,55 @@ const updateTotal = () => {
   $('#total').text(`Total: \$${(Math.round(total * 100) / 100).toFixed(2)}`);
 };
 
+const computeMileage = () => {
+  const distances = $("#lockbox_transactions input[name*='distance']:visible");
+  distances.each((index, distance) => {
+    const row = $(distance).parents('.row');
+    const amount = $(row).find('.amount-field input');
+    newAmount = $(distance).val() * 0.2;
```
Comment (BintLopez) [inline app/javascript/src/transactions.js]: how hard would it be to extract the 0.2 into an env var or yaml file?

File: lib/create_support_request.rb [hunk 2]
```diff
@@ -3,18 +3,6 @@
...
-  # lockbox_partner_id: Integer
-  # name_or_alias: String
-  # user_id: Integer
-  # client_ref_id: String
-  # lockbox_action: {
-  #   eff_date: Date,
-  #   lockbox_transactions: [
-  #     { amount: Money, category: String }
-  #   ]
-  # }
```
Comment (BintLopez) [inline lib/create_support_request.rb]: Why did you delete this? This is helpful documentation for the interface of this class.
